components/media/ImageMedia.vue

```diff
@@ -1,18 +1,18 @@
+<script lang="ts" setup>
+import { defineProps } from '#app'
+
+defineProps<{
+  src?: string
+  mimeType?: string
+}>()
```

Comment:
the type is repeated many times 

may create

```ts
type MediaType = {
   src?: string
   mimeType?: string
 }
```
for better readability

Reply:
unfortunately still not supported https://vuejs.org/guide/typescript/composition-api.html#syntax-limitations